gateway/cpanfile.snapshot

```diff
@@ -529,6 +960,60 @@ DISTRIBUTIONS
       List::Util 1.09
       Scalar::Util 1.09
       Test::Builder 0
+  Test-Harness-3.42
```

Comment:
Something does not look right. There are dependencies that were not here before, like this one.